test/integration/hotrestart_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
```

Comment:
I think it's still going to take a couple of weeks to fully remove cmake. There is still a bunch of stuff to sort out, people to ramp up, etc. If the effort is small to avoid the duplication it seems better to me. If it's complicated for some reason it's fine to just copy it.